API/app.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db/connection");
+const express = require("express");
+const { getTopics } = require("./controllers/controller");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (req, res, next) => {
+  res.status(400).send({ msg: "Bad request" });
+});
+
+app.use((err, req, res, next) => {
+  console.log(err);
```

Comment:
Make sure to take logs out before review. This is also a custom error handler, is it really needed here?